### PR TITLE
PLANET-3655 Add Makefile target to install and configure xdebug

### PR DESCRIPTION
**Pull request by kirdia**
I did not find a reliable way to get ip address in mac without introducing a new system dependency.
It uses the ip address of the first interface of the system (hopefully).
`$ ipconfig getifaddr en0`

Tried also some variants of docker macos specific dns names but did not work.
`host.docker.internal`
`docker.for.mac.localhost`
`docker.for.mac.host.localhost`
Although I could ping some of them, I could not make them connect to dbpg.
https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds

Also tried `xdebug.remote_connect_back` to get rid of `xdebug.remote_host`, but didn't work out either.

### CODE REVIEW

File: dev.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ gulp git_hooks
 composer install
 popd
 
-for plugin in blocks engagingnetworks medialibrary
+for plugin in blocks engagingnetworks medialibrary gutenberg-blocks
```
Comment (27Bslash6) [inline dev.sh]: Unrelated?

Comment (kirdia) [reply dev.sh]: Yes, I removed the change

**Comment by kirdia** [comment]
Updated

**Comment by kirdia** [comment]
Added missing command to update packages and install xdebug module.
@27Bslash6 ready for review

**Comment by 27Bslash6** [comment]
LGTM!
